nectarchain/user_scripts/ggrolleron/load_wfs_compute_charge.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+#import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+from pathlib import Path
+import sys
+import os
+
+import logging
+logging.basicConfig(format='%(asctime)s %(name)s %(levelname)s %(message)s',level=logging.INFO,filename = f"{os.environ.get('NECTARCHAIN_LOG')}/{Path(__file__).stem}_{os.getpid()}_load_wfs_charge.log")
```

Comment:
Document that a `$NECTARCHAIN_LOG` environment variable is expected ?

Reply:
It is not a mandatory variable, but I defined it in the README